calculation/seistech_calc/seistech_calc/plots/plt_plotting.py

```diff
@@ -731,11 +729,17 @@ def plot_gms_mw_rrup(
 
     Parameters
     ----------
-    metadata: Dict
+    gms_result: gms.GMSResult
     disagg_mean_values: pd.DataFrame
```

Comment:
Nice catch, is supposed to be series :joy:

Reply:
>Also, I'm not a fan of having this as a docstring if it's just repeating the type-hinting / default arguments above as it's just two things we need to keep in-sync.

This is a fair call as some docstring like this, with no extra explanation, is basically duplicate information from the function's param. E.g., def plot_gms_mw_rrup(gms_result: gms.GMSResult,....), we know that `gms_result` is an GMSResult object.

However, because we've been keeping parameters inside the docstring regardless, will keep it for now.
Perhaps, we can also have a chat about this like we did for the import guidelines.